docs/taskruns.md

```diff
@@ -184,7 +184,6 @@ spec:
     name: mytask
   podTemplate:
     schedulerName: volcano
-    hostNetwork: true
```

Comment:
was removing this intentional?

Reply:
yes, as I wrote in the PR description. Using `hostNetwork` is a "special-case" and should not be used as a general example of using PodTemplate (as what I think). But it is now documented in the Pod-template page, it was missing there.

Reply:
Missed that 😅 Thanks!